Specs/Scene/ImplicitTilesetSpec.js

```diff
@@ -26,7 +26,7 @@ describe("Scene/ImplicitTileset", function () {
       "3DTILES_implicit_tiling": {
         subdivisionScheme: "QUADTREE",
         subtreeLevels: 3,
-        maximumLevel: 4,
+        availableLevels: 4,
```

Comment:
Make sure to add a test for the older `maximumLevel` schema too, though use `availableLevels` wherever possible.

That said, don't worry about the implicit tilesets in `/Specs/Data` right now, better to wait for a later PR in this staging branch.